distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1424,54 +1447,244 @@ public void testProductDownloadListEmptyWhenNoDownloads() {
     }
 
     @Test
-    public void testProductDownloadListWithOneActiveDownload() {
+    public void testProductDownloadListWithOneActiveDownload() throws IOException {
+
         String filename = "product.txt";
         String metacardId = generateUniqueMetacardId();
         String resourceData = getResourceData(metacardId);
 
-        cswServer.whenHttp()
-                .match(post("/services/csw"),
-                        withPostBodyContaining("GetRecords"),
-                        withPostBodyContaining(metacardId))
-                .then(ok(),
-                        contentType("text/xml"),
-                        bytesContent(getCswQueryResponse(metacardId).getBytes()));
+        int delayBetweenRetries = 500;
 
-        cswServer.whenHttp()
-                .match(Condition.get("/services/csw"),
-                        Condition.parameter("request", "GetRecordById"),
-                        Condition.parameter("id", metacardId))
-                .then(getCswRetrievalHeaders(filename),
-                        chunkedContentWithHeaders(resourceData, Duration.ofMillis(500), 0));
-
-        String startDownloadUrl =
-                RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl() + "?source=" + CSW_STUB_SOURCE_ID
-                        + "&metacard=" + metacardId;
+        String startDownloadUrl = setUpDownloadResponse(metacardId, filename, resourceData, NO_RETRIES, delayBetweenRetries);
 
         // @formatter:off
         String downloadId = when().get(startDownloadUrl).then().log().all()
                 .extract().jsonPath().getString("downloadId");
         // @formatter:on
+        assertThat(downloadId, not(isEmptyString()));
 
         String getAllDownloadsUrl = RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl();
 
         // @formatter:off
-        List<Map<String, Object>> downloads = expect("List of active downloads is not empty").within(10, SECONDS)
+        List<Map<String, Object>> downloads = expect("List of active downloads is not empty").within(30, SECONDS)
                 .until(()-> when().get(getAllDownloadsUrl)
                         .then().log().all().extract().body().jsonPath().getList(""), hasSize(1)).lastResult();
         // @formatter:on
 
         Map download = downloads.get(0);
         assertThat(download.size(), is(6));
 
-        assertThat(download.get("downloadId"), is(downloadId));
-        assertThat(download.get("fileName"), is(filename));
-        assertThat(download.get("status"), is("IN_PROGRESS"));
-        int bytesDownloaded = (int) download.get("bytesDownloaded");
-        assertThat(bytesDownloaded, is(greaterThan(0)));
-        assertThat(bytesDownloaded, is(lessThan(resourceData.length() + 1)));
-        assertTrue(((String) download.get("percentDownloaded")).matches("UNKNOWN|[0-9]|[0-9]{2}|100"));
-        assertThat((List<String>) download.get("users"), contains("Guest@Guest@127.0.0.1"));
+        checkDownloadInProgress(download, downloadId, filename, resourceData);
```

Comment:
`assertDownloadInProgress` or `verifyDownloadInProgress` might be more "standard".